StandardConfig/production/ILDReconstruction.py

```diff
@@ -85,35 +91,33 @@
     default="ILD_l5_o1_v02",
 )
 parser.add_argument(
-    "--perfectPFA", help="Run perfect PandoraPFA", action="store_true", default=False
+    "--perfectPFA",
+    help="Run perfect PandoraPFA",
+    action="store_true",
 )
 parser.add_argument(
     "--runOverlay",
     help="Run background overaly. NOTE: You have to make sure that the Overlay algorithms in "
     " BgOverlay/BgOverlay.py are provided with the necessary overlay files",
     action="store_true",
-    default=False,
-)
-# BeamCal reco configuration
-parser.add_argument(
-    "--runBeamCalReco",
-    help="Run the BeamCal reco",
-    action="store_true",
-    dest="runBeamCalReco",
 )
 parser.add_argument(
     "--noBeamCalReco",
     help="Don't run the BeamCal reco",
     action="store_false",
     dest="runBeamCalReco",
 )
-parser.set_defaults(runBeamCalReco=True)
 parser.add_argument(
     "--beamCalCalibFactor",
     help="The BeamCal calibration constant from sim hit energy to calibrated calo hit energy",
     type=float,
     default=79.6,
 )
+parser.add_argument(
+    "--trackingOnly",
+    help="Only Tracking is performed; built for reco testing purposes",
+    action="store_true",
+)
```

Comment:
Does this option work with a "normal" geometry? I.e. does the reconstruction then run without exiting early or crashing?